build-system/window-config.js

```diff
@@ -0,0 +1,8 @@
+var minimist = require('minimist');
+
+var argv = minimist(process.argv.slice(2));
+var isCanary = argv.type === 'canary';
+
+exports.getTemplate = function() {
+  return `window.AMP_CONFIG={canary:${isCanary}};\n`;
```

Comment:
No line break to avoid confusing sourcemaps. Maybe add /*CONFIG*/ to make replacement easier.